db/schema.rb

```diff
@@ -1071,8 +1071,8 @@
   add_foreign_key "legal_aid_applications", "providers"
   add_foreign_key "legal_framework_merits_task_lists", "legal_aid_applications", on_delete: :cascade
   add_foreign_key "legal_framework_submissions", "legal_aid_applications"
-  add_foreign_key "linked_applications", "legal_aid_applications", column: "associated_application_id"
-  add_foreign_key "linked_applications", "legal_aid_applications", column: "lead_application_id"
+  add_foreign_key "linked_applications", "legal_aid_applications", column: "associated_application_id", validate: false
+  add_foreign_key "linked_applications", "legal_aid_applications", column: "lead_application_id", validate: false
```

Comment:
How come this has been added? This might be due to running migrations locally through rails

Reply:
Yes, I think that's what must have happened.